test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ModelBoundQuerySettings/FilterAttributeTest/FilterAttributeTest.cs

```diff
@@ -186,5 +193,27 @@ public async Task FilterSingleNavigationOrComplexProperty(string entitySetUrl, i
                 Assert.Contains("cannot be used in the $filter query option.", result);
             }
         }
+
+        [Theory]
+        [InlineData(AuthorBaseUrl + "?$filter=Books/any(d: d/BookId eq 7)")]
+        [InlineData(ModelBoundAuthorBaseUrl + "?$filter=Books/any(d: d/BookId eq 7)")]
```

Comment:
You changed codes in SelectExpandBinder, but the test doesn't refresh the changes?

Reply:
The change in `SelectExpandBinder` was to address a warning by the analyzer.
I had `if (structuredType is IEdmEntityType entityType)` and `if (structuredType is IEdmEntityType)` in the same method (introduced in my last PR) and the analyzer for the AspNet Web API project wasn't too happy about that.